SlicerCART/src/SlicerCART.py

```diff
@@ -1266,31 +1281,52 @@ def __init__(self, segmenter, modality, edit_conf, label_config_yaml = None, par
                     cell.setForeground(qt.QBrush(qt.QColor(0, 0, 0)))
                     self.label_table_view.setItem(index, 6, cell)
                     self.label_table_view.setHorizontalHeaderItem(6, qt.QTableWidgetItem('Max. HU'))
+                    
+      self.label_table_view.setSizePolicy(qt.QSizePolicy.Expanding, qt.QSizePolicy.Fixed)
 
       self.add_label_button = qt.QPushButton('Add Label')
       self.add_label_button.clicked.connect(self.push_add_label)
       layout.addWidget(self.add_label_button)
 
-      self.save_button = qt.QPushButton('Save')
-      self.save_button.clicked.connect(self.push_save)
-      layout.addWidget(self.save_button)
+      display_timer_hbox = qt.QHBoxLayout()
+
+      self.display_timer_label = qt.QLabel('Display timer during segmentation? ')
+      self.display_timer_label.setStyleSheet("font-weight: bold")
+      display_timer_hbox.addWidget(self.display_timer_label)
+
+      self.display_timer_checkbox = qt.QCheckBox()
+      display_timer_hbox.addWidget(self.display_timer_checkbox)
+
+      layout.addLayout(display_timer_hbox)
+
+      self.apply_button = qt.QPushButton('Apply')
+      self.apply_button.clicked.connect(self.push_apply)
+      layout.addWidget(self.apply_button)
 
       self.cancel_button = qt.QPushButton('Cancel')
       self.cancel_button.clicked.connect(self.push_cancel)
-      layout.addWidget(self.cancel_button)
+      layout.addWidget(self.cancel_button)      
+
+      self.populate_default_values()
+      self.connect_buttons_to_callbacks()
 
       self.setLayout(layout)
-      self.setWindowTitle("Configure Labels")
-      self.resize(800, 200)
+      self.setWindowTitle("Configure Segmentation")
+      self.resize(500, 600)
+
+   def push_add_label(self):
```

Comment:
When I click on the add label in COnfigure segmentation, the pop-up closes and I do not have any result.

Reply:
What do you mean by no result? Is the new label not saved? Or do you mean that you have to click `Configure segmentation` again to view the newly added label?

Reply:
no label and the pop up just closes.

Reply:
the reason for this issue is that I've set an input mask that forces users to enter three digit or four digit numbers depending on the chosen option (CT or MRI). This was due to issue #53. However, it seems that although it forces integers, the field is already filled with 3 or 4 spaces and prevents users from typing in giving the impression that the field is full.

Reply:
Hm I think this does not refer to the same thin i was referring to... I mean this add label button? 
![image](https://github.com/user-attachments/assets/f8e69164-45cc-4da5-b6e6-6e2276268e9d)

Reply:
I am not sure to be able to reproduce this error. I know that there a known bug that prevents the user from inputting a `value`, but it is due to #28

Reply:
ok considered. We might open another issue for it after this pull request.

Reply:
could you give me the steps you took to reach the bug you mentioned? I am able to add, delete and edit labels on my side.

Reply:
little video showing the bug https://www.dropbox.com/scl/fi/leqvfnmgsr5dl3mhacb1t/Screen-Recording-2024-10-27-at-18.08.24.mov?rlkey=fle0rzjlu1ol5b7prpr62r2ce&st=kz8ycgt5&dl=0